src/renderer/components/message/Message.tsx

```diff
@@ -289,27 +282,33 @@ function buildContextMenu(
 export default function Message(props: {
   message: Type.Message
   conversationType: ConversationType
-  /* onRetrySend */
 }) {
   const { message, conversationType } = props
   const { id, viewType, text, hasLocation, isSetupmessage, hasHtml } = message
   const direction = getDirection(message)
   const status = mapCoreMsgStatus2String(message.state)
   const tx = useTranslationFunction()
+  const accountId = selectedAccountId()
 
   const screenContext = useContext(ScreenContext)
   const { openContextMenu, openDialog } = screenContext
 
   const showMenu: (
     event: React.MouseEvent<HTMLDivElement | HTMLAnchorElement, MouseEvent>
-  ) => void = event => {
+  ) => Promise<void> = async event => {
+    const chat = await BackendRemote.rpc.getFullChatById(
+      accountId,
+      message.chatId
+    )
```

Comment:
we could probably better use the message display context for that? this might add a mini delay to the opening of context menu?

Reply:
but such things can also be in a follow up pr.

Reply:
Yes! I was wondering about that, thanks for the pointer.